mods/tnt/README.txt

```diff
@@ -23,22 +23,33 @@ All gunpowder textures except tnt_gunpowder_inventory.png.
 sofar (sofar@foo-projects.org) (CC BY-SA 3.0):
 tnt_blast.png
 
+paramat (CC BY-SA 3.0)
+tnt_tnt_stick.png
+
 Introduction
 ------------
 This mod adds TNT to Minetest. TNT is a tool to help the player
 in mining.
 
 How to use the mod:
+
 Craft gunpowder by placing coal and gravel in the crafting area.
-The gunpowder can be used to craft TNT or as fuse for TNT.
-To craft TNT place items like this:
--- wood - gunpowder -- wood -
-gunpowder gunpowder gunpowder
--- wood - gunpowder -- wood -
+The gunpowder can be used to craft TNT sticks or as a fuse trail for TNT.
+
+To craft 2 TNT sticks:
+G_G
+GPG
+G_G
+G = gunpowder
+P = paper
+The sticks are not usable as an explosive.
+
+Craft TNT from 9 TNT sticks.
 
-There are different ways to blow up TNT:
+There are different ways to ignite TNT:
   1. Hit it with a torch.
-  2. Hit a gunpowder fuse that leads to a TNT block with a torch or flint-and-steel.
+  2. Hit a gunpowder fuse trail that leads to TNT with a torch or
+     flint-and-steel.
   3. Activate it with mesecons (fastest way).
 
-Be aware of the damage radius of 6 blocks!
+Be aware of the player damage radius of 6 blocks!
```

Comment:
Not limited to players. All mortal objects will take damage, unless on_blast is defined.
`blocks` -> `nodes` (or metres)

Reply:
Ok.